### DISCUSSION TITLE
Github-Linkedin-&-Email

### DISCUSSION THREAD

**Discussion by bgoonz** [bgoonz/lambda-prep#16] [General]
Add your contact info below for future reference & networking!

**Comment by bgoonz** [comment]
[Github](https://github.com/bgoonz)
[Linkedin](https://www.linkedin.com/in/bryan-guner-046199128/)
[Email](bryan.guner.dev@gmail.com)

**Reply by Ethan-Edmond** [reply]
[Github](https://github.com/Ethan-Edmond)
[LinkedIn](https://www.linkedin.com/in/ethan-edmond-a02b62186)
email is eedmond@uccs.edu

**Reply by Michael-Kochis** [reply]
[Github](https://github.com/Michael-Kochis)
[LinkedIn](https://www.linkedin.com/in/michael-kochis-43316623/)
[Email](mailto:rhishisikk@yahoo.com)

**Reply by topdude15** [reply]
[GitHub](https://github.com/topdude15)
[LinkedIn](https://www.linkedin.com/in/trevor-rose-65a043171/)
E-Mail is trose@leapaheadstudios.com
